src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -21,6 +23,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], help='List of URLs/CIDRs for which proxy should not to be used.')
+        c.argument('proxy_cert', options_list=['--proxy-cert'], help='Path to the certificate file for proxy')
```

Comment:
Why not add `type=file_type, completer=FilesCompleter()` here?

Reply:
Thanks for pointing it out. Have added it